style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+  overflow: hidden;
+}
+
+:root {
+  --base: #ffc600;
+  --spacing: 10px;
+  --blur: 0px;
+  --grayscale: 0%;
+  --invert: 0%;
+  --saturate: 100%;
+  --hue-rotate: 0;
+  --sepia: 0%;
+}
+
+img {
+  padding: var(--spacing);
+  background: var(--base);
+  filter: blur(var(--blur)) grayscale(var(--grayscale)) invert(var(--invert)) saturate(var(--saturate)) hue-rotate(var(--hue-rotate)) sepia(var(--sepia));
+  margin: 0 5%;
+  width: 100%;
+}
+
+.hl {
+  color: var(--base);
+}
+
+h2 {
+  text-align: center;
+  margin-bottom: 5%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  background: #193549;
+  color: white;
+  font-family: "helvetica neue", sans-serif;
+  font-weight: 100;
+  font-size: 20px;
+}
+
+.container {
+  margin: 0 100px;
+  align-items: center;
+  align-content: center;
```

Comment:
How does this property works in this case? Why do you need it here?

Reply:
@KiStanislav you haven't checked align-content here... Do you still need it here? what is the role of this property?

Reply:
DONE!